ibr/ibr-sdk/src/index.js

```diff
@@ -117,36 +117,57 @@ function createLabel(tagName, name, parentTag, forId = undefined) {
   return label;
 }
 
+/**
+ * Create checkboxes for given visualization.
+ * @param {Object} visualization the visualization object to be
+ added to the sidebar.
+ * @param {String} visualizationName the name of the visualization being
+ processed.
+ * @param {String} structureName the name of parent structure of the
+ visualization.
+ */
+function createCheckboxForVisualization(visualization, visualizationName,
```

Comment:
we chatted about index.js being just imports. could we use this opportunity to create a file that contains the render functions and put this one in there?

ultimately our goal for this file is to index the package exports (see: Three.js)